parsers/gdscript.c

```diff
@@ -0,0 +1,784 @@
+/*
+*   Copyright (c) 2000-2003, Darren Hiebert
+*
+*   This source code is released for free distribution under the terms of the
+*   GNU General Public License version 2 or (at your option) any later version.
+*
+*   This module contains functions for generating tags for GDScript language
+*   files.
```

Comment:
If you know, could add a URL for the GDScript language reference here?

Reply:
Yeah sure